runtime/pretty/print.go

```diff
@@ -1,32 +1,17 @@
-/*
- * Cadence - The resource-oriented smart contract programming language
```

Comment:
Just curious why this file doesn't get the copyright header?

Reply:
Good catch! I copied the code over to a new file and Git detected the move. While copying I forgot to copy the license header. There was a bug in the license header check script, I fixed it in 
2459ac3 and added the missing license headers in 7ea2ea4